IFC4x3/Properties/d/DrillingDiameter_32JL8Vw$L9BhN7Dgy0bzPU/DocProperty.xml

```diff
@@ -0,0 +1,2 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Is this really needed as a property? Quantity or extracted from geometry should be quite okay here.

Reply:
What if the representation is just a line? Ping @j-weil.

Reply:
Understood. I would like to keep this in mind (unless documented somewhere else) that we need a valid representation usage for 'Axis' for `IfcBorehole`.

Reply:
We haved added a usage for this.